transformato/analysis.py

```diff
@@ -611,6 +636,7 @@ def calculate_dG_to_common_core(
         num_proc: int = 1,
         in_memory: bool = False,
         nr_of_max_snapshots: int = -1,
+        consecutive_runs: int = 0,
```

Comment:
Yes, the runs always start with 1 (run1,...):

https://github.com/wiederm/transformato/blob/744a0e010b416f27b611128aed13b0b9c588ac05/transformato/analysis.py#L447

`0` is just used when `multiple_runs` is not used instead of `False`